monkey/infection_monkey/ransomware/utils.py

```diff
@@ -0,0 +1,96 @@
+import os
+from typing import List
+
+VALID_FILE_EXTENSIONS_FOR_ENCRYPTION = {
+    ".3ds",
+    ".7z",
+    ".accdb",
+    ".ai",
+    ".asp",
+    ".aspx",
+    ".avhd",
+    ".avi",
+    ".back",
+    ".bak",
+    ".c",
+    ".cfg",
+    ".conf",
+    ".cpp",
+    ".cs",
+    ".ctl",
+    ".dbf",
+    ".disk",
+    ".djvu",
+    ".doc",
+    ".docx",
+    ".dwg",
+    ".eml",
+    ".fdb",
+    ".giff",
+    ".gz",
+    ".h",
+    ".hdd",
+    ".jpg",
+    ".jpeg",
+    ".kdbx",
+    ".mail",
+    ".mdb",
+    ".mpg",
+    ".mpeg",
+    ".msg",
+    ".nrg",
+    ".ora",
+    ".ost",
+    ".ova",
+    ".ovf",
+    ".pdf",
+    ".php",
+    ".pmf",
+    ".png",
+    ".ppt",
+    ".pptx",
+    ".pst",
+    ".pvi",
+    ".py",
+    ".pyc",
+    ".rar",
+    ".rtf",
+    ".sln",
+    ".sql",
+    ".tar",
+    ".tiff",
+    ".txt",
+    ".vbox",
+    ".vbs",
+    ".vcb",
+    ".vdi",
+    ".vfd",
+    ".vmc",
+    ".vmdk",
+    ".vmsd",
+    ".vmx",
+    ".vsdx",
+    ".vsv",
+    ".work",
+    ".xls",
+    ".xlsx",
+    ".xvd",
+    ".zip",
+}
+
+
+def get_files_to_encrypt(dir_path: str) -> List[str]:
```

Comment:
I think it's worth a bit of investment to make file listing more extendable as it might be useful for the exfiltration scenario in the future.

I'd propose to make a function which would take a list of directories and a list of filters. Then, it would gather all files in provided directories and apply all filters. The output would be a list of files that pass the filters.

Reply:
I feel like that's overkill for now. And we also didn't estimate keeping this in mind.